siobrultech_protocols/gem/packets.py

```diff
@@ -239,6 +245,44 @@ def parse(self, packet: bytes) -> Packet:
         return Packet(**args)  # type: ignore
 
 
+class ECMPacketFormat(PacketFormat):
```

Comment:
Should this exist at a different path since it's not for the GEM?

Reply:
Eh, the directory naming may not have been the best, but I don't think there's a lot of value in moving this to a completely separate directory.

Reply:
Fair.  Then before we cut a new release, let's get rid of `gem` and we can make this 1.0 since I think it's the last thing we really needed to support anyway.

Reply:
Oh that's a good point; do you want a release cut with the changes so far, or do you plan to add the API as well?

Reply:
I'd like to do another 0.x release with this support. I'm not sure when I'll get around to the ECM API, honestly. I'm in the process of updating my package and Home Assistant so that the API isn't a requirement, since DashBoxes and some other setups make it not work anyway.